### ISSUE TITLE
Python: No hightlighting for 'not' and 'in'

### ISSUE THREAD

**Issue by Galarzaa90** [rokoroku/vscode-theme-darcula#10]
Seems like highlighting is missing for these keywords in Python.

Here's how it looks on vscode darcula:
![image](https://user-images.githubusercontent.com/12865379/39482580-77887e8a-4d24-11e8-8d00-873ef2532eb1.png)

Here's how it looks on pycharm's darcula:
![image](https://user-images.githubusercontent.com/12865379/39482629-a824145a-4d24-11e8-816d-ab07cc04de41.png)

And for reference, here's vscode Dark+:
![image](https://user-images.githubusercontent.com/12865379/39482668-d5f4d374-4d24-11e8-8f6a-9241d02755ff.png)

**Comment by richardchien** [comment]
There is also no highlighting for `is`.

**Comment by agubelu** [comment]
You can highlight them by adding this to your personal preferences (Ctrl + comma):

```js
"editor.tokenColorCustomizations": {
    "[Darcula]": {
        "textMateRules": [
            {
                "scope": "keyword.operator.logical",
                "settings": {
                    "foreground": "#CC8242"
                }
            },
        ],
    }
},
```

**Comment by rokoroku** [comment]
Fixed in `v1.1.3` via 629eebd. Thanks @agubelu !
